.github/workflows/performance.yml

```diff
@@ -30,6 +30,15 @@ concurrency:
 permissions: {}
 
 env:
+  # Environment variables for Docker and PHP-FPM configuration
+  PHP_FPM_UID: ${{ steps.get-php-fpm-uid.outputs.php-fpm-uid }}
```

Comment:
Sorry, I think my initial feedback was confusing. these environment variables should remain as they were unchanged.